content/ios/getting-started.md

```diff
@@ -12,66 +12,54 @@ This guide will show you how to integrate Skygear into an existing iOS project.
 
-1. Open and edit the `Podfile` file, add the following in the last line:
+### Step 1: Install SDK using CocoaPods
 
-```
-pod "SKYKit"
-```
+To install the Skygear iOS SDK as your iOS application dependency:
 
-2. Run `pod install` in your terminal.
-3. It's done! You now have installed Skygear SDK in your app.
+1. You need to close Xcode.
+2. Open Terminal and navigate to the directory that contains your iOS
+   project by using the cd command: `cd ~/Path/To/Your/App`.
+   You can just drag the file icon to Terminal and the path will be 	automatically typed for you.
+3. Run this command in Terminal `pod init` to create a [Podfile](https://guides.cocoapods.org/using/the-podfile.html).
+4. Run this command in Terminal `open -a Xcode Podfile` to edit the Podfile using Xcode. You should avoid using TextEdit to edit the Podfile because it may mess up the format and confuse CocoaPods.
 
-### For Swift Projects
+### Step 2: Edit the Podfile
 
 1. Open and edit the `Podfile` file. Your `Podfile` file should look like this:
 
 ```
 use_frameworks!
 platform :ios, '8.0'
-pod 'SKYKit'
+
+target 'YourProjectName' do
+    pod 'SKYKit'
+end
 ```
 
-Cocoapods 0.36 and above introduces the `use_frameworks!` instruction, so the Objective-C bridging header is no longer needed.
+Replace 'YourProjectName' with your actual project name.
+
+CocoaPods 0.36 and above introduces the `use_frameworks!` instruction, so the Objective-C bridging header is no longer needed if you're using Swift in your project.
 
 2. Run `pod install` in your terminal.
 3. You would see that an Xcode Workspace file is created. Open the file and go to the project.
-4. It's done! You have installed Skygear SDK in your app. Just import the SDK in each swift file to call the SDK.
+4. It's done! You have installed Skygear SDK in your app. If you're using Swift, just import the SDK in each Swift file to call the SDK.
 
-<a name="set-up-ios-app"></a>
-## Setting up your iOS app
+### Step 3: Configure container
 
-`SKYContainer` is the uppermost layer of `SKYKit`. It represents the root of all
-resources accessible by an application and one application should have exactly
-one container. In `SKYKit`, such container is accessed via the singleton
-`defaultContainer`:
-
-```obj-c
-SKYContainer *container = [SKYContainer defaultContainer];
-```
-
-Container provides [User Authentication]({{< relref "user.md" >}}),
-[Asset Storage]({{< relref "asset.md" >}}) and access to
-[public and private databases]({{< relref "#SKYDatabase" >}}).
-
-Now, you are going to setup the server endpoint and API key for your app.
-
-The `SKYContainer` (Skygear Container) object is the primary interface for interacting with the Skygear service. The object is initialized with the sever endpoint and an API key.
+Now, you are going to setup the server endpoint and API key for your app. Read mroe about SKYContainer [here](#skycontainer).
```

Comment:
typo for "more". Also avoid using "here" as the link text, you can directly make `SkyContainer` a link.